packages/sdk/src/chains/EVM/__test__/depositFns.test.ts

```diff
@@ -89,11 +87,69 @@ describe('deposit functions', () => {
       expect(bridgeInstance.deposit).toHaveBeenCalledTimes(1);
     });
 
+    it('should successfully execute deposit without overrides', async () => {
```

Comment:
I'm not sure that the test name is describes what is being tested, basically this checks that `bridgeInstance` which is wrapped inside `executeDeposit` is called with the values passed to `executeDeposit`

Reply:
Yes, that's correct. I've renamed the test.